test/testlib/nuodb_database_utilities.go

```diff
@@ -820,13 +820,27 @@ type BackupHookResponse struct {
 	Message string `json:"message"`
 }
 
-func GetBackupHookResponse(t *testing.T, namespaceName, smPod, resourcePath string) *BackupHookResponse {
+func GetBackupHookResponse(t *testing.T, namespaceName, smPod, resourcePath string, payload ...string) *BackupHookResponse {
```

Comment:
Why not pass a map that gets serialized to JSON?

Reply:
I didn't want to change all the callers of this function. This is why I used a variadic argument.

Reply:
Okay, thanks for the explanation.